yb-voyager/src/migassessment/common.go

```diff
@@ -19,12 +19,14 @@ import (
 	"encoding/csv"
 	"encoding/json"
 	"fmt"
-	log "github.com/sirupsen/logrus"
 	"net/http"
 	"os"
+
+	log "github.com/sirupsen/logrus"
 )
 
 var AssessmentMetadataDir string
```

Comment:
Can we remove this now from this file?
or are we using AssessmentMetadataDir somewhere else?